java-squid/src/main/java/org/sonar/java/JavaClasspath.java

```diff
@@ -53,6 +53,9 @@ protected void init() {
         libraries = getFilesFromProperty("sonar.libraries");
       }
       elements = Lists.newArrayList(binaries);
+      if(libraries.isEmpty()) {
+        LOG.warn("Bytecode of dependencies was not provided for analysis of source files, you might end up with less precise results. Bytecode can be provided using sonar.java.libraries property");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 197 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)